generator/integration_tests/tests/golden_thing_admin_rest_stub_test.cc

```diff
@@ -114,8 +118,12 @@ TEST(GoldenThingAdminRestStubTest, ListDatabases) {
   EXPECT_THAT(success->next_page_token(), Eq("my_next_page_token"));
 }
 
-TEST(GoldenThingAdminRestStubTest, CreateDatabase) {
-  auto mock_rest_client = absl::make_unique<MockRestClient>();
+TEST(GoldenThingAdminRestStubTest, AsyncCreateDatabase) {
+  auto mock_cq = std::make_shared<MockCompletionQueueImpl>();
+  EXPECT_CALL(*mock_cq, RunAsync)
```

Comment:
If you want, you could delay this execution until later (maybe using `AsyncSequencer`) to verify your code handles slow thread starts correctly.